src/cleanup_rules/swift/edges.toml

```diff
@@ -38,22 +52,22 @@ to = [
 
 [[edges]]
 scope = "Parent"
-from = "replace_expression_with_boolean_literal"
-to = ["boolean_literal_cleanup", "variable_inline_cleanup"]
+from = "replace_identifier_with_value"
```

Comment:
is this change overlapping with the other PR?

Reply:
Yeah, this is because nature of newly added tests are such. Landing either of PRs would solve it for all.

Reply:
hmmmm. Ok. But u will have to rebase the other one,